src/Tizen.System.Session/Session/SessionEnums.cs

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+using System.ComponentModel;
+
+namespace Tizen.System
+{
+    internal enum SessionEventType : int
```

Comment:
```suggestion
    [Flags]
    internal enum SessionEventType : int
```
https://learn.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-7.0

Reply:
Thanks for hints. Unfortunately this case is more complicated, as the structure actually contains unions, which in turn constist of another structures:

	subsession_event_type_e event;
	int session_uid;

	union {
		struct {
			subsession_user_t user;
		} add_user;

		struct {
			subsession_user_t user;
		} remove_user;

		struct {
			uint64_t switch_id;
			subsession_user_t prev_user;
			subsession_user_t next_user;
		} switch_user;
	};
	
We've tried to use LayoutKind Explicit for unions and LayoutKind.Sequential for structs, but this unfortunately didn't work properly, event with a proper explicitly stated alignments.